index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+  <body>
+    <header>
+        <nav>
+            <div class="toolbar">
+                <div>
+                    <a href="">Ishmael</a> 
+                </div>
+                <div>
+                    <img src="img/union.png" alt="">
+                </div>
+            </div>
```

Comment:
- [ ] Well done on implementing the change requested by the first reviewer. However, a new problem happened after making the change. 
![image](https://user-images.githubusercontent.com/43698511/138177308-41d97cee-87da-4210-a664-18f0a148dd5d.png)

- [ ] Kindly align the logo with the menu icon